_data/sites.json

```
@@ -1268,9 +1268,9 @@
         "name": "AniList",
         "url": "https://anilist.co/settings/account",
         "difficulty": "easy",
```

```suggestion
        "difficulty": "medium",
```
Issue stated that it is now medium and I agree, since you need to come back to the website 24h later.